x-pack/solutions/observability/plugins/serverless_observability/public/navigation_tree.ts

```diff
@@ -355,10 +355,73 @@ export const createNavigationTree = ({
         breadcrumbStatus: 'hidden',
         children: [
           {
-            link: 'management',
+            id: 'management',
             title: i18n.translate('xpack.serverlessObservability.nav.mngt', {
               defaultMessage: 'Management',
             }),
+            spaceBefore: null,
+            renderAs: 'panelOpener',
+            children: [
+              {
+                title: i18n.translate('xpack.serverlessObservability.nav.mngt.data', {
+                  defaultMessage: 'Data',
```

Comment:
@elastic/kibana-management it looks like this structure has a more official source of truth within the Management plugin, but it is not accessible to outside plugins [1]. Is there a chance we could update the public start contract of the Management plugin so I wouldn't need this declaration? I am worried this information could be missing a critical link, or become out of date. 

[1] Per: https://github.com/elastic/kibana/blob/2fd8994/src/platform/plugins/shared/management/public/management_sections_service.ts#L51

Reply:
@tsullivan I'm sure this can be improved but I think the code is a good representation of the group dynamics that produced it - stateful kibana has a way of expressing management app listing and solutions want to control their own listing completely independent of whatever stateful is doing. We could certainly expose some management content but its unclear to me how this would be helpful.

Reply:
Thanks @mattkime. I have recently taken over this code and I wanted to make sure that there isn't an obvious alternative to this implementation that would be better. It sounds like keeping solutions in control of their own listing is what we should keep doing.

> stateful kibana has a way of expressing management app listing

It could be pointed out that independent control isn't just an issue for serverless. The Solution Nav is used in stateful deployments when the Space uses a Solution view, and the listing of management apps is completely controlled by the solution there as well. That goes back prior to this PR, though.